modules/sdk-coin-zeta/src/lib/constants.ts

```diff
@@ -2,3 +2,5 @@ export const validDenoms = ['azeta'];
 export const accountAddressRegex = /^(zeta)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;
 export const validatorAddressRegex = /^(zetavaloper)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;
 export const contractAddressRegex = /^(zeta)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']+)$/;
+export const GAS_LIMIT = 250000;
+export const GAS_AMOUNT = '125000';
```

Comment:
what is gas amount?

Reply:
gas fee for the transaction

Reply:
nit: gas_amount looks confusing, fee_amount or fee_estimate would be better

Reply:
Also, what's the rationale for having one be a string and another one be a number between `GAS_LIMIT` AND `GAS_AMOUNT`

Reply:
```
export interface GasAmountDetails {
  gasAmount: string;
  gasLimit: number;
}
```

🙃

Reply:
@margueriteblair 

this was intentional;
1. This is the common controller code uses which references the individual coin's `gasAmount` and `gasLimit` - https://github.com/BitGo/BitGoJS/blob/d5c6c74845e62ee3cd68321c55d729da1526fcae/modules/abstract-cosmos/src/cosmosCoin.ts#L171-L174
2. The `FeeData` type used above - https://github.com/BitGo/BitGoJS/blob/master/modules/abstract-cosmos/src/lib/iface.ts#L56-L59 contains mixed types and not just pure strings or pure numbers